### PR TITLE
buildingore update

### PR DESCRIPTION
**Pull request by mabafaba**
also changed some other stuff dunnnoooo oh oh

### CODE REVIEW

File: vignettes/detailed_example.Rmd [hunk 1]
```diff
@@ -49,7 +51,7 @@ parent <- data.frame(uuid=1:10, age=sample(30:60,10,replace = T),parent_gender=s
 child <- data.frame(parent_uuid=sample(1:10,20,replace = T), age=sample(20,1,18), child_gender=sample(c("F","M"),20,replace = T))
 ```
 
-## Affect parent age to each child
+## Attach parent age to each child
```
Comment (sharonorengo) [inline vignettes/detailed_example.Rmd]: this should not change unless function names are changed as well

File: vignettes/detailed_example.Rmd [hunk 2]
```diff
@@ -58,7 +60,7 @@ family <- add_parent_to_loop(child,parent,uuid.name.loop = "parent_uuid",uuid.na
 
 run `?add_parent_to_loop` for details.
 
-## Affect all parent information to each child
+## Attach all parent information to each child
```
Comment (sharonorengo) [inline vignettes/detailed_example.Rmd]: same as above

File: .Rbuildignore [hunk 3]
```diff
@@ -3,4 +3,4 @@
 ^koboloops\.Rproj$
 ^\.Rproj\.user$
 ^.*\.Rproj$
-*.Rhistory
\ No newline at end of file
+^.*\.Rhistory$
```
Comment (sharonorengo) [inline .Rbuildignore:6]: WOW youre a genius

**Review by sharonorengo — CHANGES_REQUESTED** [review]

File: vignettes/detailed_example.Rmd (hunk 2 above)
Comment (mabafaba) [reply vignettes/detailed_example.Rmd]: reverted this back! thanks

File: vignettes/detailed_example.Rmd (hunk 1 above)
Comment (mabafaba) [reply vignettes/detailed_example.Rmd]: reverted this back! thanks

File: .Rbuildignore (hunk 3 above)
Comment (mabafaba) [reply .Rbuildignore:6]: thanks i know!

**Review by mabafaba — COMMENTED** [review]
I addressed all comments

**Review by sharonorengo — APPROVED** [review]

**Pull request merged**